components/script/lib.rs

```diff
@@ -6,7 +6,6 @@
 #![feature(core_intrinsics)]
 #![feature(drain_filter)]
 #![feature(inner_deref)]
-#![feature(on_unimplemented)]
```

Comment:
If we use the rustc_attrs feature, can we keep using rustc_on_unimplemented?

Reply:
I haven’t tried, but the wording of the error message seems fairly strong in suggesting we shouldn’t? It would also not-prevent the use of many other attributes.